FirebaseRemoteConfig/Sources/FIRRemoteConfig.m

```diff
@@ -236,27 +236,28 @@ - (void)fetchAndActivateWithCompletionHandler:
     (FIRRemoteConfigFetchAndActivateCompletion)completionHandler {
   __weak FIRRemoteConfig *weakSelf = self;
   FIRRemoteConfigFetchCompletion fetchCompletion =
-      ^(FIRRemoteConfigFetchStatus fetchStatus, NSError *error) {
+      ^(FIRRemoteConfigFetchStatus fetchStatus, NSError *fetchError) {
```

Comment:
This method is a great candidate to be refactored with Promises. We can consider it for future PRs.

Reply:
Agreed! This and several other RC methods.